docs/deployment/cloud.rst

```diff
@@ -54,35 +53,26 @@ Docker: An easy option for local development
 
 This section will describe the steps to build and deploy the infrastructure with **Docker**.
 
-If you have ``make`` available you can run the next commands in your terminal:
-
-.. code-block::
-   :caption: run the commands from the root of the project
-
-        $ make build-all
-
-If you do not have ``make`` available (Windows for example), you can build the images manually:
-
-.. code-block::
-   :caption: run the commands from the root of the project
-
-        $ docker build -t qiskit/quantum-serverless-notebook -f ./infrastructure/docker/Dockerfile-notebook .
-        $ docker build -t qiskit/quantum-serverless-ray-node -f ./infrastructure/docker/Dockerfile-ray-qiskit .
-        $ ...
-
-Now that you have built the necessary Docker images you can run **docker-compose** to deploy the project locally:
+If you have ``docker-compose`` available you can run the next command in your terminal:
```

Comment:
we should update all these to `docker compose` (without the hyphen) -- `docker-compose` (with the hyphen) is being sunset in the near future

Reply:
Done in [4667b6a](https://github.com/Qiskit-Extensions/quantum-serverless/pull/868/commits/4667b6aebdc1afe85a17a76a5431ee54f4537f02)